src/pages/Resources/Resources.js

```diff
@@ -58,6 +67,18 @@ function Resources() {
           </Typography>
         )}
         <br />
+        <Pagination
```

Comment:
UI nitpick: what do you think about moving the pagination component to the bottom of the grid? :)